README.md

```diff
@@ -12,30 +15,107 @@ is simpler and easier to work with. Additionally, for developers familar with
 the Go programming language will be instantly comfortable with Gunk files and
 syntax.
 
+### Contents
+
+* [How it works](#how-it-works)
+* [Features](#features)
+* [Installing](#installing)
+* [Usage](#usage)
+* [Contributing](#contributing)
+
+## How it works
+
+Gunk works by using Go's standard library `go/ast` package (and related
+packages) to parse all `*.gunk` files in a "Gunk Package" (and applying the
+same kinds of rules as Go package), building the appropos protobuf messages. In
+turn, those are passed to `protoc-gen-*` tools, along with any passed options.
+
+## Features
+
+- Written in simple and idiomatic [Go][go-project]
+- Intuitive and [easy to use](#usage)
+- Works on Linux, Mac and Windows
+
 ## Installing
 
-Gunk can be installed in the usual Go fashion:
+Gunk requires [Go][go-project] to be installed on your machine. To install Go,
+follow the [installation instructions][go-install] for your specific operating
+systems.
+
+Gunk can then be installed in the usual Go fashion:
+
+	$ go get -u github.com/gunk/gunk
 
-	go get -u github.com/gunk/gunk
+## Usage
 
-## Syntax
+### Syntax
 
 The aim of Gunk is to provide Go-compatible syntax that can be natively read
 and handled by the `go/*` package. As such, Gunk definitions are a subset of
 the Go programming language:
 
 	// examples/util/echo.gunk
 
-## Working with `*.gunk` files
+### Working with `*.gunk` files
 
 Working with the `gunk` command line tool should be instantly recognizable to
 experienced Go developers:
 
-	gunk generate ./examples/util
+	$ gunk generate ./examples/util
 
-## Overview
+### More information
 
-Gunk works by using Go's standard library `go/ast` package (and related
-packages) to parse all `*.gunk` files in a "Gunk Package" (and applying the
-same kinds of rules as Go package), building the appropos protobuf messages. In
-turn, those are passed to `protoc-gen-*` tools, along with any passed options.
+Please see [the GoDoc API page][godoc] for a
+full API listing.
+
+## Contributing
+
+### Bug Reports & Feature Requests
+
+Please use the [issue tracker][issue-tracker] to report any bugs or file feature requests.
+
+### Developing
+
+PRs are welcome. To begin developing, do this:
+
+```bash
+$ git clone git@github.com:gunk/gunk.git
```

Comment:
https please - otherwise you force the user to have an SSH key set up with a github account.